src/languages/typescript.js

```diff
@@ -28,10 +28,18 @@ export default function(hljs) {
     "unknown"
   ];
   const NAMESPACE = {
-    beginKeywords: 'namespace',
+    begin: [
+      /namespace/,
+      /\s+/,
+      hljs.IDENT_RE
+    ],
+    beginScope: {
+      1: "keyword",
+      3: "variable"
```

Comment:
From the examples (https://www.typescriptlang.org/docs/handbook/namespaces.html) it looks like namespaces are class-like and CamelCase in their naming... so I think `title.class` would be more appropriate here than variable.